localstack/services/sns/sns_starter.py

```diff
@@ -1,5 +1,31 @@
 from localstack import config
 
+PUBLISH_BATCH_TEMPLATE = """<PublishBatchResponse xmlns="http://sns.amazonaws.com/doc/2010-03-31/">
```

Comment:
Rather than the template based approach that `moto` uses, we usually construct the responses manually, as this reduces duplication and also allows us to utilize some of our util functions like [`requests_response_xml(..)`](https://github.com/localstack/localstack/blob/10a51f8a09146125970ba49122704d6f1faf1985/localstack/utils/aws/aws_responses.py#L126-L141). Some examples of how this is being used can be found in the codebase, e.g., in `cloudformation_api.py`.

Also, we are soon going to migrate to a new way of defining the service interfaces using code generation - an example can be found for SQS here: https://github.com/localstack/localstack/blob/10a51f8a09146125970ba49122704d6f1faf1985/localstack/aws/api/sqs/__init__.py#L681-L687 This will allow us to use fully typed request and response messages - again, we'll construct the response messages there, rather than using the template based approach.

If it's not too much hassle, it would be great if we could refactor the PR to remove the template, and construct the result via `requests_response_xml(..)` instead.. Thanks!